### PR TITLE
s1023025

### PR DESCRIPTION
**Pull request by s1023025**

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,10 +1,7 @@
 #include <stdio.h>
 
 int main() {
-	printf("1022997\n");
-	printf("1023008\n");
-	printf("1024869\n");
-	printf("1023006\n");
```
Comment (jrjang2) [inline whoami.c]: Do not modify the original code.

**Comment by jrjang2** [comment]
Merge all commits to one before doing pull request.

**Comment by s1023025** [comment]
I am done

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

File: whoami.c [hunk 2]
```diff
@@ -9,6 +9,8 @@ int main() {
 	printf("1023007\n");
 	printf("1023003\n");
 	printf("1023030\n");
+    printf("1023025\n");
```
Comment (jrjang2) [inline whoami.c]: Remove this redundant line.

**Comment by s1023025** [comment]
i have done

**Comment by jrjang2** [comment]
Good. But there is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by s1023025** [comment]
i have done

**Comment by jrjang2** [comment]
Good. But there is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by s1023025** [comment]
i am OK

**Comment by jrjang2** [comment]
Good!

**Pull request merged**